From c5b914df81f2c3c7c97ce98a1a5ff7c6c088756a Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Mon, 27 Feb 2017 17:44:40 -0800
Subject: [PATCH] joyent/node-cueball#87 documentation refers to DNSResolver,
 but this is not exported in index.js Reviewed by: Joshua M. Clulow
 <jmc@joyent.com>

---
 CHANGES.adoc | 9 +++++++++
 lib/index.js | 1 +
 package.json | 2 +-
 3 files changed, 11 insertions(+), 1 deletion(-)

diff --git a/CHANGES.adoc b/CHANGES.adoc
index d92829d..f24f900 100644
--- a/CHANGES.adoc
+++ b/CHANGES.adoc
@@ -6,6 +6,15 @@ toc::[]
 
 ## v2.x
 
+### v2.2.0
+
+New minor release, due to addition of new API.
+
+API changes:
+
+ - The `DNSResolver` class is now exported in the package root, as specified
+   in the documentation. It was mistakenly left out.
+
 ### v2.1.1
 
 Maintenance release.
diff --git a/lib/index.js b/lib/index.js
index dcbdd57..28a9b1c 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -20,6 +20,7 @@ module.exports = {
 	ConnectionPool: mod_pool.ConnectionPool,
 	ConnectionSet: mod_cset.ConnectionSet,
 	Resolver: mod_resolver.Resolver,
+	DNSResolver: mod_resolver.DNSResolver,
 	StaticIpResolver: mod_resolver.StaticIpResolver,
 	resolverForIpOrDomain: mod_resolver.resolverForIpOrDomain,
 
diff --git a/package.json b/package.json
index d526295..d5523c4 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cueball",
-  "version": "2.1.1",
+  "version": "2.2.0",
   "description": "manage a pool of connections to a multi-node service where nodes are listed in DNS",
   "main": "lib/index.js",
   "dependencies": {
-- 
2.21.0

